bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitAccessoryUpdater.java

```diff
@@ -63,6 +64,28 @@ public void subscribe(GenericItem item, String key, HomekitCharacteristicChangeC
         });
     }
 
+    public void subscribeToUpdates(GenericItem item, String key, Consumer<State> callback) {
+        logger.trace("Received subscription request for {} / {}", item, key);
```

Comment:
do we need all these logging? i know, it is at trace level and helps for debugging but we dont have such detailed logging nowhere.

i would suggest to remove all logging which does not help end user, e.g. if callback is null then it is probably a bug and fine to skip it.

Reply:
Debug and trace are usually there to… debug if something unexpected occurs.
Those or not logged by default, so end users don’t see them. Same as other comment how can trace be annoying ?

Reply:
if i want to troubleshoot something and enable logging in trace, i get a lot of getKeyFromMapping which is not really  useful. it says only the function got executed.   

same for subscribeToUpdates - it will add at least 2 lines of every subscription, so, a lot of logging. 

but i would agree to keep to for this version as it is new function and we might need to troubleshoot something on other users' setup. we could remove it in the next release once we see it works as expected

Reply:
I've dealt with issues in the past where the trace logging on subscriptions was very important. I feel like if you're enabling TRACE logging, you're kind of signing up for a firehose, and should be well versed to know how to filter down to exactly what you need (by only enabling for a single class, or if you enable for an entire bundle, knowing how to silence particularly noisy classes that are irrelevant to your current investigation).